configs/encapsulate_in_connect.yaml

```diff
@@ -0,0 +1,38 @@
+admin:
```

Comment:
drive by: please make sure these new tests are added to the BUILD file so they get tested as part of config_test. Thank you!